scipy/spatial/tests/conftest.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+
+from scipy.spatial.distance import _METRICS_NAMES
+
+
+@pytest.fixture(params=_METRICS_NAMES, scope="session")
+def metric(request):
```

Comment:
Can't this be defined in `test_distance.py` since it's the only file that needs it? That might help with the meson failures as well.

Reply:
That fixture should be defined wherever `_METRICS_NAMES` is defined. I can move the whole thing to `test_distance.py` though.

Reply:
Moved to `test_distance.py` now.